### PR TITLE
Add project_directory configuration variable. Fixes #5.

### LINKED ISSUE #5: Question to name the directory different from the package name
At the moment, when you run cookiecutter and enter a project_name it becomes used for both the directory and the python package instead.

### PR DESCRIPTION
**Pull request by glogiotatidis**
Add project_directory configuration variable to seperate project
directory from python package name.

### CODE REVIEW

**Comment by peterbe** [comment]
https://github.com/mozilla/sugardough/pull/14 has now been merged. Hint hint.

**Comment by glogiotatidis** [comment]
ack. i'll update this, this week.

**Comment by peterbe** [comment]
Would you mind rebasing this when #34 is done so we have a clean test? Then I'll look into it more deeply.

**Comment by glogiotatidis** [comment]
@peterbe rebased!

**Comment by peterbe** [comment]
Why no README in sugardough_dir?

**Comment by peterbe** [comment]
Would it be possible to call it `project-dir` instead? Since by putting a `-` in the name you're indicating that it's never going to be a Python thing.

**Comment by glogiotatidis** [comment]
Updated with `project-dir` and double checked that all files got moved to the new directory.

File: {{ cookiecutter.project_directory }}/README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-{{ cookiecutter.project_name }}
+{{ cookiecutter.package_name }}
 ==========
 
-[![Build Status](https://img.shields.io/travis/mozilla/{{ cookiecutter.project_name }}/master.svg)](https://travis-ci.org/mozilla/{{ cookiecutter.project_name }})
+[![Build Status](https://img.shields.io/travis/mozilla/{{ cookiecutter.package_name }}/master.svg)](https://travis-ci.org/mozilla/{{ cookiecutter.package_name }})
```
Comment (peterbe) [inline {{ cookiecutter.project_directory }}/README.md]: Uh? Shouldn't this be `project_directory`?

**Pull request closed without merging**